backend/app/services/execution/agents/base_intent_analyzer.py

```
@@ -0,0 +1,53 @@
+            base_url=model_config.get("base_url"),
+        )
+
+        try:
+            response = await client.chat.completions.create(
+                model=model_config.get("model_id", "gpt-4o-mini"),
+                messages=[{"role": "user", "content": prompt}],
+                temperature=0,
+                response_format={"type": "json_object"},
+            )
+            return json.loads(response.choices[0].message.content)
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)